docker/compose_files/docker-compose.yml

```diff
@@ -106,6 +106,12 @@ services:
     <<: *codalab-base
     depends_on:
       - rest-server
+      - bundle-manager
+      - frontend
+      - nginx
+      - mysql
```

Comment:
mysql is technically a dependency, but mysql should be a dependency of rest-server and bundle-manager

Reply:
I updated the code and split all the dependencies for each service.